src/sim-host/ui/custom-elements.js

```diff
@@ -375,12 +388,19 @@ function initialize(changePanelVisibilityCallback) {
             }
         },
         initialize: function () {
-            this.shadowRoot.querySelector('label').textContent = this.getAttribute('label');
+            var displayLabel = this.getAttribute('label');
+            this.shadowRoot.querySelector('label').textContent = displayLabel;
             this.classList.add('cordova-panel-row');
             this.classList.add('cordova-group');
             this._internalValue = 0;
 
             var input = this.shadowRoot.querySelector('input');
+            var readLabel = this.getAttribute('spoken-text');
+            if (readLabel) {
+                input.setAttribute('aria-label', readLabel);
+            } else {
+                input.setAttribute('aria-label', displayLabel);
+            }
```

Comment:
You can replace the above few lines with a single line:

    input.setAttribute('aria-label', this.getAttribute('spoken-text') || displayLabel);